### PR TITLE
Running format on files and updating spacing on touched files

### PR DESCRIPTION
**Pull request by escobarjonatan**

### CODE REVIEW

**Comment by HenryQuillin** [comment]
Let's discuss using theme.spacing vs the default number values here: https://projectnewmgroup.slack.com/archives/C033DLCBGSK/p1658108292107289

**Review by HenryQuillin — CHANGES_REQUESTED** [review]
Let's discuss using theme.spacing vs the default number values here: https://projectnewmgroup.slack.com/archives/C033DLCBGSK/p1658108292107289

File: src/components/Toast.tsx [hunk 1]
```diff
@@ -37,20 +35,24 @@ const Toast = () => {
...
+          success: <CheckCircleIcon />,
         } }
         onClose={ handleClose }
         severity={ severity }
         sx={ {
           "&.MuiAlert-root": {
             ".MuiAlert-message": {
-              paddingRight: ["8px", "16px", "48px"]
+              paddingRight: [
+                theme.spacing(1),
```
Comment (HenryQuillin) [inline src/components/Toast.tsx:52]: I think you forgot to remove the theme.spacing here. Not a big deal tho

File: src/components/Toast.tsx [hunk 2]
```diff
@@ -37,20 +35,24 @@ const Toast = () => {
...
-          success: <CheckCircleIcon />
+          success: <CheckCircleIcon />,
         } }
         onClose={ handleClose }
         severity={ severity }
         sx={ {
           "&.MuiAlert-root": {
             ".MuiAlert-message": {
-              paddingRight: ["8px", "16px", "48px"]
+              paddingRight: [
```
Comment (HenryQuillin) [inline src/components/Toast.tsx:51]: Going forward, I think we should use either p, pr, pl... or padding, paddingRight, paddingLeft.... My vote is for the former for brevity.

**Review by HenryQuillin — APPROVED** [review]
LGTM!

File: src/components/Toast.tsx (hunk 1 above)
Comment (escobarjonatan) [reply src/components/Toast.tsx:52]: there are soooooo many ways to do the same thing lol i didnt know you could use the numbers just like that, i actually prefer the numbers as well.... will be changing it soon

File: src/components/Toast.tsx (hunk 2 above)
Comment (escobarjonatan) [reply src/components/Toast.tsx]: We can definitely talk about it, these are the cons of us having the flexibility to do it multiple ways :/

**Pull request merged**